lib/vsc/fancylogger.py

```diff
@@ -39,7 +39,7 @@
 
 # it's ok if this fails with only vsc-install installed.
 # we cannot introduce a dependency on vsc-base (and we do not care)
-from vsc.utils.fancylogger import *
+from vsc.utils.fancylogger import *  # noqa: F403
```

Comment:
I still would just remove this.

Reply:
it will be a separate PR. See https://jira.ugent.be/browse/HPC-7616